backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/ResourceRegistry/PaginatedList.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Altinn.AccessManagement.UI.Core.Models.ResourceRegistry
+{
+    /// <summary>
+    /// One page of resources
+    /// </summary>
+    public class PaginatedList<T>
+    {
+        /// <summary>
+        /// Current page
+        /// </summary>
+        public int Page { get; set; }
+
+        /// <summary>
+        /// Total number of list entires
+        /// </summary>
+        public int NumEntriesTotal { get; set; }
+
+        /// <summary>
+        /// The list entries pertaining to the current page
+        /// </summary>
+        public List<T> PageList { get; set; }
+
+        /// <summary>
+        /// Parameterless constructor
+        /// </summary>
+        public PaginatedList()
```

Comment:
Why do you need the Parameterless constructor?

Reply:
Because json deserialization needs one to work (or at least it complained that it did before I added it)

Reply:
But you have another constructor below it. In which file do you get the error?

Reply:
Yes, but the other constructor is not parameterless, and apparently that is what deserialization requires. It is the same with ServiceResourceFE where I also added a parameterless constructor because the tests failed when it didn't have one